dataherald/api/__init__.py

```diff
@@ -71,9 +73,11 @@ def get_scanned_databases(self, db_alias: str) -> ScannedDBResponse:
         pass
 
     @abstractmethod
-    def delete_golden_record(self, golden_record_id: str) -> bool:
+    def delete_golden_record(self, namespace: str, golden_record_id: str) -> dict:
```

Comment:
What are we supplying for namespace? is it the organization id? name? db_alias?

Reply:
@DishenWang2023 it should be organization name or id. it doesn't make any difference. In the tests that I put in the docs, I used dataherald.

Reply:
I think the db_alias (if it is unique) is the right choice

Reply:
It's not enforced to be unique.